packages/concerto-vocabulary/lib/vocabularymanager.js

```diff
@@ -333,7 +333,7 @@ class VocabularyManager {
                                 }
                             });
                         }
-                        else if(term.localeCompare('properties')) {
+                        else if(term === 'properties') {
```

Comment:
Hi @abhicodes01 
Thanks for you interest.
Can you please explain in detail what is this change about? Is there any issue associated with this?